README.rst

```diff
@@ -34,6 +34,14 @@ This library uses API token to authenticate you. To generate or access your API
     from amigocloud import AmigoCloud
     amigocloud = AmigoCloud(token='R:dlNDEiOWciP3y26kG2cHklYpr2HIPK40HD32r1')
 
+You could also use a project token.
```

Comment:
_Perhaps_ make it clear that project tokens are project-scoped ? (As that is my current understanding) Further below there is an example showing that you can access a /users url with a project token and this may generate confusion.